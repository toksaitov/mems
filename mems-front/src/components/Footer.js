import React from 'react';

const Footer = () =>
    <footer>
        <nav className="navbar navbar-light bg-light">
            <p className="mx-auto mt-3">&copy; <i>DEVCIT</i> 2021</p>
        </nav>
    </footer>;

export default Footer;
