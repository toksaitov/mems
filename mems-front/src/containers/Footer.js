import Footer from '../components/Footer.js';

export default Footer;
