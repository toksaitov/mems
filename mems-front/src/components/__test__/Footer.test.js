import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Footer from '../Footer.js';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer />, div);
});

it('matches the previous snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
});
