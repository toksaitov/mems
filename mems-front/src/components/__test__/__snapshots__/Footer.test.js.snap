// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the previous snapshot 1`] = `
<footer>
  <nav
    className="navbar navbar-light bg-light"
  >
    <p
      className="mx-auto mt-3"
    >
      Â© 
      <i>
        DEVCIT
      </i>
       2021
    </p>
  </nav>
</footer>
`;
